programming-core/src/main/java/org/objectweb/proactive/core/httpserver/ClassServerServlet.java

```diff
@@ -132,7 +132,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         if (logger.isDebugEnabled()) {
             final String from = req.getLocalAddr() + ":" + req.getLocalPort();
             final String status = ok ? "OK" : "Class Not Found";
-            logger.debug("Served request from " + from + " for " + className + " status: " + status);
+            if (logger.isDebugEnabled()) {
```

Comment:
It seems this block is already guarded by `if (logger.isDebugEnabled()) {`

Reply:
right, I didn't see it